spec/lib/fastly_nsq/fake_message_queue_spec.rb

```diff
@@ -10,6 +10,23 @@
     end
   end
 
+  describe '@@logger' do
+    after do
+      FakeMessageQueue.logger = Logger.new(nil)
+    end
+
+    it 'is initalized as an empty Ruby Logger' do
+      expect(FakeMessageQueue.logger).to be_a Logger
+    end
+
+    it 'is settable via logger=' do
```

Comment:
Maybe also add "... and is retrievable"? We are testing both `.logger=` and `.logger`.